src/Cake.Common.Tests/Unit/XML/XmlPeekTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.IO;
+using Cake.Common.Tests.Fixtures;
+using Xunit;
+
+namespace Cake.Common.Tests.Unit.XML
+{
+    public sealed class XmlPeekTests
```

Comment:
This should be called `XmlPeekAliasesTests`.

Would prefer it you use the same naming conventions as other tests:
  `XmlPeekAliasesTests` -> `TheXmlPeekMethod` -> `Should_Throw_If_FilePath_Is_Null`